### ISSUE TITLE
Strange autocomplete behavior workaround proposal

### ISSUE THREAD

**Issue by bronius** [sequelpro/sequelpro#3077]
tl;dr; Please consider inserting as the first item of autocomplete/auto-suggest options the very thing the user typed so that when the suggested completion is not desired, the first one suggested (exactly what was typed) is the one selected by default.

In our office and on the forums, it sounds like autocomplete is just a bit quirky.  Here's an example:
<img width="167" alt="sequel pro" src="https://user-images.githubusercontent.com/2348390/41241146-deb9f964-6d61-11e8-9b24-164d240b409d.png">
I'm typing `substr(fieldname, 0, 5...` and Sequel Pro is like, "Oh maybe you are trying to write either `50` or `57`!"  (Really?.. how..?..) Well, rather than digging into the algorithm, please consider adding a new suggest item as the top/first item: `5` (or whatever is currently to be replaced).  I think this might elegantly overcome a lot of people's frustrations and be a simple fix.  Those who are hampered by autocomplete will simply get what they typed, those who see a good suggested can cursor down to the right one (usually just one down).

Thoughts?

**Comment by dmoagx** [comment]
> Those who are hampered by autocomplete will simply get what they typed
In what way are you hampered?

**Comment by dmoagx** [comment]
No feedback from reporter in > 1 month, closing.
